src/index.js

```diff
@@ -116,6 +141,8 @@ const ipnsSubdomain = (url) => isIpns(url, fqdnPattern, fqdnProtocolMatch, fqdnH
 
 module.exports = {
   multihash: isMultihash,
+  multiaddr: isMultiaddr,
+  peerMultiaddr: isPeerMultiaddr,
```

Comment:
Not for this PR but I'm musing on naming, I would ideally like to do something like `const { isMultiaddr } = require('is-ipfs')` or `import { isMultiaddr } from 'is-ipfs'` rather than have to import the whole library.

If we eventually start using es-modules we can tree shake the stuff we're not using but as it is now we're encouraging importing the whole library since (for example) `const { cid } = require('is-ipfs')` would likely conflict with cid instance vars we have in code and it's not descriptive to validate a cid like `if (cid(myCid))`.

Reply:
This is a good point, extracted into https://github.com/ipfs/is-ipfs/issues/28